components/Footer.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <footer>
+    <div class="column-left">
+      <img
+        src="~/assets/white-logo.svg"
+        alt="white witnet-logo"
+        class="footer-logo"
+      />
+      <p class="footer-description">
+        {{ this.$t('footer-description') }}
+      </p>
+      <p class="footer-description">
+        {{ this.$t('footer-copyright') }}
+      </p>
+    </div>
+    <div class="column-right">
+      <div class="social-bar">
+        <a href="https://twitter.com/witnet_io" target="_blank">
+          <img class="social" src="~/assets/twitter.svg" alt="social" />
+        </a>
+        <a href="https://discord.gg/X4uurfP" target="_blank">
+          <img class="social" src="~/assets/discord.svg" alt="social" />
+        </a>
+        <a href="https://medium.com/witnet" target="_blank">
+          <img class="social" src="~/assets/medium.svg" alt="social" />
+        </a>
+        <a
+          href="mailto:info@witnet.foundation?subject=Witnet project"
+          targer="_blank"
+        >
+          <img class="social" src="~/assets/mail.svg" alt="social" />
+        </a>
+        <a href="https://t.me/witnetio" target="_blank">
+          <img class="social" src="~/assets/telegram.svg" alt="social" />
+        </a>
+        <a href="https://github.com/witnet" target="_blank">
+          <img class="social" src="~/assets/github.svg" alt="social" />
```

Comment:
I would add a more descriptive `alt` attribute
```suggestion
          <img class="social" src="~/assets/github.svg" alt="Github logo" />
```